components/idp-mgt/org.wso2.carbon.idp.mgt/src/main/java/org/wso2/carbon/idp/mgt/listener/IDPMgtAuditLogger.java

```diff
@@ -39,6 +43,11 @@ public class IDPMgtAuditLogger extends AbstractIdentityProviderMgtListener {
             " | Result : %s ";
     private final String SUCCESS = "Success";
 
+//    Properties with following key values will not be printed in logs.
+    private static final Set<String> UNLOGGABLE_PARAMS = Collections.unmodifiableSet(
+            new HashSet<>(Arrays.asList("ClientSecret", "SPNPassword", "APISecret", "scim2-password", "sf-password",
+                    "sf-client-secret", "scim-password", "scim-default-pwd", "scim2-default-pwd")));
```

Comment:
do we have a git issue to track the improvement of not having hard-coded values here?

Reply:
[https://github.com/wso2-enterprise/asgardeo-product/issues/13686](https://github.com/wso2-enterprise/asgardeo-product/issues/13686)